gitpod-io/website
# [ref-arch/eks] correct autoscaler helm service account settings

## Description

While the external-dns and cert-manager helm charts use a top level `serviceAccount` variable, the autoscaler helm chart shifts these values under an `rbac` top level value. This issue may have been masked during testing due to IAM policies granting cluster autoscaler IAM permissions to the whole nodegroups.

This commit updates the EKS single-cluster and PoV guides accordingly.

## Related Issue(s)

## How to test

Create an EKS cluster based on the guides, install the cluster autoscaler, and check for errors emitted from the autoscaler.

## Release Notes

```release-note
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2734"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
